### PR TITLE
fix(input-group): unnest text class

### LINKED ISSUE #2286: Unnest .pf-c-input-group__text in .pf-c-input-group
Let's move `.pf-c-input-group__text` out of `.pf-c-input-group` to lower the specificity, and move `label.pf-c-input-group__text` into `.pf-c-input-group__text`

### PR DESCRIPTION
**Pull request by SpyTec**
fixes #2286

Not sure how to move `label` into `.pf-c-input-group__text` as the order needs to be `label.pf-c-input-group__text`

cc @mcoker

### CODE REVIEW

File: src/patternfly/components/InputGroup/docs/code.md [hunk 1]
```diff
@@ -13,5 +13,5 @@ When using the `.pf-c-input-group` always ensure labels are used outside the inp
 
 | Class | Applied to | Outcome |
 | -- | -- | -- |
-| `.pf-c-input-group` | `<div>` |  Iniaties the input group. **Required** |
-| `.pf-c-input-group__text` | `<span>` |  Iniaties the input group text. This can be used to show text, radio, icons, or check boxes. |
\ No newline at end of file
+| `.pf-c-input-group` | `<div>` |  Initiates the input group. **Required** |
```
Comment (mcoker) [inline src/patternfly/components/InputGroup/docs/code.md]: nice find!

File: src/patternfly/components/InputGroup/input-group.scss [hunk 2]
```diff
@@ -87,17 +84,23 @@
...
+  display: flex;
+  align-items: center;
+  padding-right: var(--pf-c-input-group__text--PaddingRight);
+  padding-left: var(--pf-c-input-group__text--PaddingLeft);
+  font-size: var(--pf-c-input-group__text--FontSize);
+  color: var(--pf-c-input-group__text--Color);
+  text-align: center;
+  background-color: var(--pf-c-input-group__text--BackgroundColor);
+  border: var(--pf-c-input-group__text--BorderWidth) solid;
+  border-color: var(--pf-c-input-group__text--BorderTopColor) var(--pf-c-input-group__text--BorderRightColor) var(--pf-c-input-group__text--BorderBottomColor) var(--pf-c-input-group__text--BorderLeftColor);
```
Comment (mcoker) [inline src/patternfly/components/InputGroup/input-group.scss]: You can add this in the `.pf-c-input-group__text` selector to support `label.pf-c-input-group__text`

```
  @at-root label#{&} {
    cursor: pointer;
  }
```

Comment (SpyTec) [reply src/patternfly/components/InputGroup/input-group.scss]: Learning something new everyday!

Thanks, I'll fix that up :)

**Comment by SpyTec** [comment]
Fixed, sorry for the late response

**Pull request merged**